config/clients/java/template/Java/libraries/native/README.mustache

```diff
@@ -0,0 +1,189 @@
+# {{artifactId}}
```

Comment:
Delete

Reply:
Thanks -- Deleted in [3dbbaf7](https://github.com/openfga/sdk-generator/pull/158/commits/3dbbaf752f74a14f3535e48df92b4a934822699d)

Reply:
Removed in [6f9804d](https://github.com/openfga/sdk-generator/pull/158/commits/6f9804d21724edeff04096b82f2a999ab7a75144)